api/src/main/java/org/pmiops/workbench/notebooks/LeonardoNotebooksClientImpl.java

```diff
@@ -137,14 +140,29 @@ public void createRuntime(
 
     DbUser user = userProvider.get();
     DbWorkspace workspace = workspaceDao.getRequired(workspaceNamespace, workspaceFirecloudName);
+    DbCdrVersion cdrVersion = workspace.getCdrVersion();
     Map<String, String> customEnvironmentVariables = new HashMap<>();
     // i.e. is NEW or MIGRATED
     if (!workspace.getBillingMigrationStatusEnum().equals(BillingMigrationStatus.OLD)) {
       customEnvironmentVariables.put(
           WORKSPACE_CDR_ENV_KEY,
-          workspace.getCdrVersion().getBigqueryProject()
-              + "."
-              + workspace.getCdrVersion().getBigqueryDataset());
+          cdrVersion.getBigqueryProject() + "." + cdrVersion.getBigqueryDataset());
+    }
+
+    if (cdrVersion.getHasMergedWgsData()) {
+      customEnvironmentVariables.put(
+          MERGED_WGS_BUCKET_KEY,
+          cdrVersion.getAccessTier().getDatasetsBucket()
+              + cdrVersion.getCdrVersionId()
+              + "/wgs/vcf/merged/");
```

Comment:
I feel like this might be something we want to be able to control from the config file. What if we just provided the full path from there?

The other benefit I see is that the path isn't split up between two parts of the code base, making it easier to look up what this value is.

Also, it would make the cdrVersion booleans unnecessary. We can just provide the variable if they exist.

Reply:
I guess I didn't want to duplicate the end of the path in the config file - but that's kind of what config files are for, heh

Reply:
looks like this still needs to be swapped out with the config variable?

Also, I think it would be good to add some defensive checks to strip a trailing slash from the bucket, add a leading slash to the bucket path, and combine them with a hard coded slash. This should prevent any potential errors from users updating the config with or without a slash in the path.

Reply:
argh! thank you